content/help/handle-split-sites.md

```diff
@@ -0,0 +1,32 @@
+# "Split" Sites: Connect Your Smart Inverter for Solar Energy Forecasting
+
+If you have a "split" site with multiple sets of solar panels pointing in different directions or tilted at different angles, there are two methods you can use to enter site details for solar energy forecasting:
+
+## Method 1: Create a Separate Site for Each Direction/Tilt
+
+The first method involves creating a new site for each differing direction or tilt. This method is useful if you want to track the performance of each set of solar panels separately. To use this method, follow these steps:
+
+1. Add a new site for each set of solar panels with a different direction or tilt.
+2. Enter the direction and tilt for each site accurately.
+3. Enter the module capacity and inverter capacity for each site.
+
+This method may result in more work for you, as you'll need to track the performance of each site separately.
+
+## Method 2: Connect Your Smart Inverter
+
+The preferred method for "split" sites is to connect your smart inverter with our app. Our app trains forecasts based on actual data returned by smart inverters, so this will get you the best forecasts regardless of tilt and direction. However, this method only works if you have smart inverters.
+
+To use this method, follow these steps:
+
+1. Add a single site for your "split" site.
+2. Enter the average tilt and direction for all sets of solar panels.
+3. Enter the module capacity and inverter capacity for the site.
+4. Connect your smart inverter with our app.
+
+This method will save you time, as you'll only need to track the performance of one site. However, it's important to note that this method only works if you have smart inverters.
+
+## Troubleshooting
+
+Here are some common issues users may encounter when trying to enter site details, and tips for how to overcome them:
+
+- I'm not sure how to average out the tilts and directions: If you're unsure how to average out the tilts and directions for your
```

Comment:
idk if i'm missing something, but is there more to this text? also, if you can, add more troubleshooting bullets